source/time.tex

```diff
@@ -1335,7 +1335,7 @@
 The defaulted copy constructor of duration shall be a
 constexpr function if and only if the required initialization
 of the member \tcode{rep_} for copy and move, respectively, would
-satisfy the requirements for a constexpr function.
+meet the requirements for a constexpr function.
```

Comment:
This one should be "satisfy". See [dcl.constexpr]/3,4,6 and [class.copy.ctor]/12.